CIP-1904/README.md

```diff
@@ -0,0 +1,250 @@
+---
+CIP: 1904
+Title: Verifiable supply chain records
```

Comment:
I want to instead keep this broad and not limited to proof of origin, since it can cover more than we did.